.gitignore

```diff
@@ -1 +1,3 @@
-_site
\ No newline at end of file
+_site
+Gemfile.lock
```

Comment:
According to the official doc of bundler, `Gemfile.lock` should be versioned (they insist quite a lot of that). Could you keep in it under git's control?